.gitmodules

```diff
@@ -1,4 +1,8 @@
 [submodule "admin/tool/externaltaskmonitor"]
-	path = admin/tool/externaltaskmonitor
-	url = https://github.com/ucsf-education/tool_externaltaskmonitor
-	branch = MOODLE_404_STABLE
+    path = admin/tool/externaltaskmonitor
```

Comment:
Hey @stopfstedt, does your `git submodule add` command insert spaces for indentation instead of tabs? Did you have to change any git configuration to do that? Mine is defaulted to insert tabs. Please lmk.

Reply:
reverted back to tabs in https://github.com/ucsf-education/moodle/pull/68